ldclient/client.py

```diff
@@ -156,18 +108,19 @@ def __init__(self, api_key, config=None):
         self._offline = False
```

Comment:
One thing we did in the other refactorings like this is change offline mode to a configuration parameter rather than a modifiable state. This makes it a lot easier to reason about things, and it makes it the case that we don't have to lazily initialize the streaming connection.